packages/messenger-widget/src/components/MessageInputBox/bl.ts

```diff
@@ -97,6 +102,42 @@ const handleNewUserMessage = async (
     const haltDelivery = getHaltDelivery(state);
     const sendDependencies: SendDependencies = getDependencies(state, account);
 
+    try {
+        const { encryptedEnvelop } = await buildEnvelop(
+            messageData,
+            encryptAsymmetric,
+            sendDependencies,
+        );
+        const envelopSize = getEnvelopSize(encryptedEnvelop);
+        const sizeLimit = state.cache.messageSizeLimit;
+
+        if (envelopSize > sizeLimit) {
+            dispatch({
+                type: UiViewStateType.SetMessageView,
+                payload: {
+                    actionType: MessageActionType.NONE,
+                    messageData: undefined,
+                },
+            });
+
+            openErrorModal(
+                'The size of the message is larger than limit '
+                    .concat(sizeLimit.toString(), ' bytes. ')
+                    .concat('Please reduce the message size.'),
+                false,
+                closeErrorModal,
+            );
+
+            return;
+        }
+    } catch (error) {}
```

Comment:
Please log at least something to indicate the error

Reply:
ok